xml/System.Reflection.Metadata.Ecma335/MetadataBuilder.xml

```diff
@@ -20,7 +20,23 @@
   <Interfaces />
   <Docs>
     <summary>The MetadataBuilder class reads and writes metadata for an assembly in a highly performant manner. It is designed for use by compilers and other assembly generation tools.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+## Remarks
+The <xref:System.Reflection.Metadata.Ecma335.MetadataBuilder> class enables you to programmatically generate assemblies. These assemblies can be saved to a file, unlike dynamic assemblies generated by <xref:System.Reflection.Emit.AssemblyBuilder> class, which does not support saving assemblies to a file on .NET 5 and .NET Core. 
+
+The `MetadataBuilder` API operates low-level metadata constructs, such as tables or blobs. For a simpler way to generate assemblies dynamically using C#, see <xref:Microsoft.CodeAnalysis.CSharp.CSharpCompilation> in Roslyn API.
+
+The format of CLI metadata is defined by the ECMA-335 specification. For more information, see [Standard ECMA-335 - Common Language Infrastructure (CLI)](http://www.ecma-international.org/publications/standards/Ecma-335.htm) on the Ecma International Web site.
+
+## Examples
+This example shows how to emit a console application assembly using <xref:System.Reflection.Metadata.Ecma335.MetadataBuilder>:
+
+[!code-csharp[](~/samples/snippets/csharp/api/system.reflection.metadata.ecma335/metadatabuilder/MetadataBuilderSnippets.cs#EmitConsoleApp)]
```

Comment:
I see different format used for adding example files, this might be better especially if you want ot add another example file (`Program.cs`)
https://github.com/dotnet/dotnet-api-docs/blob/11242b9ba428e75dcebb3e166c462fdcf2449493/xml/System.Reflection/ExceptionHandlingClause.xml#L71
```suggestion
:::code language="csharp" source="~/samples/snippets/csharp/api/system.reflection.metadata.ecma335/metadatabuilder/MetadataBuilderSnippets.cs#EmitConsoleApp" id="Snippet1":::
```

Reply:
I think i fixed the snippet syntax, but CI build wasn't picking the last changes for some reason. Could you check if the latest commit renders snippet properly?

Reply:
Looks good, thanks!